### PR TITLE
CAMEL-20684: Improve MicroProfile Fault Tolerance timer setup

### PR DESCRIPTION
**Pull request by jamesnetherton**

### CODE REVIEW

File: components/camel-microprofile/camel-microprofile-fault-tolerance/src/main/java/org/apache/camel/component/microprofile/faulttolerance/FaultToleranceTimerService.java [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+import io.smallrye.faulttolerance.core.timer.Timer;
+import org.apache.camel.CamelContext;
+import org.apache.camel.CamelContextAware;
+import org.apache.camel.Service;
+
+/**
+ * Service to manage the lifecycle of the SmallRye Fault Tolerance Timer. Primarily used when running without CDI
+ * container support.
+ */
+public class FaultToleranceTimerService implements Service, CamelContextAware {
```
Comment (davsclaus) [inline components/camel-microprofile/camel-microprofile-fault-tolerance/src/main/java/org/apache/camel/component/microprofile/faulttolerance/FaultToleranceTimerService.java:31]: Its better to extend ServiceSupport and use doStart / doStop

**Review by davsclaus — APPROVED** [review]

**Review by oscerd — APPROVED** [review]

File: components/camel-microprofile/camel-microprofile-fault-tolerance/src/main/java/org/apache/camel/component/microprofile/faulttolerance/FaultToleranceTimerService.java (hunk 1 above)
Comment (jamesnetherton) [reply components/camel-microprofile/camel-microprofile-fault-tolerance/src/main/java/org/apache/camel/component/microprofile/faulttolerance/FaultToleranceTimerService.java]: Thanks for the hint. I switch to using `ServiceSupport`.

**Pull request merged**